pythonforandroid/recipes/py3dns/__init__.py

```diff
@@ -0,0 +1,13 @@
+from pythonforandroid.recipe import PythonRecipe
+
+
+class Py3DNSRecipe(PythonRecipe):
+    site_packages_name = 'DNS'
+    version = '3.2.1'
+    url = 'https://github.com/Neizvestnyj/py3dns/archive/{version}.zip'
```

Comment:
How about using the official launchpad download link instead?
https://launchpad.net/py3dns/trunk/3.2.1/+download/py3dns-3.2.1.tar.gz
```suggestion
    url = 'https://launchpad.net/py3dns/trunk/{version}/+download/py3dns-{version}.tar.gz'
```

Reply:
Hmm, well, I didn't think about it. [py3dns](https://pypi.org/project/py3dns/) works fine and does not require any changes yet. It's just that there is such a bug on android. The last `validate_email` release was in 2015 :)